src/transport/AdminPairingTable.cpp

```diff
@@ -136,5 +259,21 @@ void AdminPairingTable::Reset()
     }
 }
 
+CHIP_ERROR AdminPairingTable::Init(PersistentStorageDelegate * storage)
+{
+    VerifyOrReturnError(storage != nullptr, CHIP_ERROR_INVALID_ARGUMENT);
+    gStorage = storage;
```

Comment:
should probably be a member variable rather than a global.

Reply:
See my comments above ^^ in regards to : https://github.com/project-chip/connectedhomeip/issues/6419

Reply:
With my latest commit:
- all Load/Store/Delete calls go through the AdminPairingTable which can be retrieved via GetGlobalPairingTable
- the AdminPairingInfo Load/Store/Delete calls take in a storage parameter once again and are only exposed to the friend class AdminPairingTable
- The persisted storage object from Server.cpp is not a global anymore but private memeber of AdminPairingTable 
- The AdminPairingTableDelegate is also a private member now. It is marked with a TODO to remove in the future once AdminPairingTable stores everything in the right spot